tools.py

```diff
@@ -509,17 +511,58 @@ def split(
 		indent = 2
 	)
 
-#transcript = json.load(open('data/transcripts_valset_16082020.csv.json_GreedyDecoder.json'))
+# transcript = json.load(open('data/transcripts_valset_16082020.csv.json_GreedyDecoder.json'))
 #
-#cer = float(torch.FloatTensor([t['cer'] for t in transcript if t['num_words'] > 0]).mean())
-#wer = float(torch.FloatTensor([t['wer'] for t in transcript if t['num_words'] > 0]).mean())
-#print('base', cer, wer)
+# cer = float(torch.FloatTensor([t['cer'] for t in transcript if t['num_words'] > 0]).mean())
+# wer = float(torch.FloatTensor([t['wer'] for t in transcript if t['num_words'] > 0]).mean())
+# print('base', cer, wer)
 #
-#for k in ['words_only_proper', 'words_only_number']:
+# for k in ['words_only_proper', 'words_only_number']:
 #	cer = float(torch.FloatTensor([t[k]['cer'] for t in transcript if t[k]['num_words'] > 0]).mean())
 #	wer = float(torch.FloatTensor([t[k]['wer'] for t in transcript if t[k]['num_words'] > 0]).mean())
 #	print(k, cer, wer)
 
+
+'''
+This script helps to find solution for input and output shapes of signal with frontend divisibility restrictions
+
+example:
+python tools.py find_solution_for_frontend_input_output_shapes_divisibility --start 119 --end 121 --input-shape-restriction 16 --output-shape-restriction 32
+'''
+
+def find_solution_for_frontend_input_output_shapes_divisibility(
+		window_size,
+		window_stride,
+		sample_rate,
+		start,
+		end,
+		input_shape_restriction,
```

Comment:
please change the arg name to something more specific, e..g `input_time_dim_multiple`

Reply:
done